app/mailers/notification_mailer.rb

```diff
@@ -11,6 +11,11 @@ def daily_summary_email(addressee, summary_report = nil)
     mail(to: addressee.email, subject: t('.subject', date: Date.yesterday))
   end
 
+  def daily_auctions_broadcast_email(recipient:, auctions:)
```

Comment:
User is not a recipient, still. One may think there is a `Recipient` role (=class), but here is none. By my previous suggestion I meant passing user's email as a `recipient` value. Is it really needed to pass the whole object here?

Reply:
How does mailer know which locale to use if it's not aware of user and it's locale?

Reply:
Btw thanks for bringing this up, because just discovered it doesn't send email in et locale anyways at the moment :)

Reply:
> How does mailer know which locale to use if it's not aware of user and it's locale?

You need just user's locale, not `I18n.locale` right? Then one option is pass the whole object `user` (`def daily_auctions_broadcast_email(user:)`, like you did initially, just with the different param name).

P.S.

> is it really needed to pass the whole object here?

I just asked whether is it needed to pass it, not that you did it wrong :)

Reply:
P.S. II: example of using "built-in" translation if you need to pass some data to translation value https://github.com/internetee/registry/blob/master/app/mailers/invoice_mailer.rb#L5. Not sure if you need it in this case, but this is a good way to use standard translation keys.